logging_demo/test/test_logging_demo.py.in

```diff
@@ -38,12 +38,9 @@ def generate_test_description(ready_fn):
 
 
 class TestLoggingDemo(unittest.TestCase):
-
```

Comment:
You might want to keep this empty line?

Reply:
Done in https://github.com/ros2/demos/pull/400/commits/9056e2d1932c10558b5e37e6ad2b60b96d1dfab8